ivy/stateful/module.py

```diff
@@ -973,3 +879,233 @@ def load(filename):
         if ivy.current_backend_str() == "paddle":
             loaded._convert_numpy_to_tensors()
         return loaded
+
+
+class PaddleIvyModule(Module):
```

Comment:
these classes should be made private, because someone being able to use `ivy.PaddleIvyModule` would lead to confusion given that `ivy.Module` is what we want users to use. Same applies to the other classes